src/components/tableV2/action.js

```diff
@@ -49,25 +50,42 @@ const Action = ({
           actionButtonDirection={actionButtonDirection}
           declineLabel={declineLabel}
           confirmLabel={confirmLabel}
-          title={confirmationTitle}
-          message={confirmationMessage}
+          title={
+            typeof confirmationTitle === "function"
+              ? confirmationTitle(currentRow?.original)
+              : confirmationTitle
+          }
+          message={
+            typeof confirmationMessage === "function"
+              ? confirmationMessage(currentRow?.original)
+              : confirmationMessage
+          }
           handleDecline={onActionDeclined}
           handleConfirm={onActionConfrimed}
         />
       )}
       <Tooltip content={tooltipText}>
         <Flex
-          data-testid={`netdata-table-action-${id}${testPrefix}`}
           alignItems="center"
           justifyContent="center"
-          height={"100%"}
-          _hover={{ background: "borderSecondary" }}
-          cursor="pointer"
+          _hover={{ background: disabled ? null : "borderSecondary" }}
+          cursor={disabled ? "auto" : "pointer"}
           key={id}
-          width={10}
-          onClick={onActionClicked}
+          round
+          padding={[1, 0]}
         >
-          <Box as={Icon} name={icon} />
+          <IconButton
+            iconSize="small"
+            data-testid={`netdata-table-action-${id}${testPrefix}`}
+            disabled={disabled}
+            onClick={onActionClicked}
+            icon={icon}
+            flavour="borderless"
+            sx={{
+              width: "16px",
+              height: "18px !important",
```

Comment:
Is `!important` really needed here?

Reply:
@burbuli8ra styles were getting override from main button component afair